research/src/fetch_campaign_donations.py

```diff
@@ -0,0 +1,346 @@
+donations_committees_concatenated.rename(columns={'Descrição da receita': 'revenue_description'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Entrega em conjunto?': 'batch'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Espécie recurso': 'type_of_revenue'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Fonte recurso': 'source_of_revenue'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Município': 'city'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome candidato': 'candidate_name'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome da UE': 'electoral_unit_name'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome do doador': 'donor_name'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome do doador (Receita Federal)': 'donor_name_for_federal_revenue'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome do doador originário': 'original_donor_name'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Nome do doador originário (Receita Federal)': 'original_donor_name_for_federal_revenue'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número candidato': 'candidate_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número candidato doador': 'donor_candidate_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número do documento': 'document_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número partido doador': 'donor_party_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número Recibo Eleitoral': 'electoral_receipt_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Número UE': 'electoral_unit_number'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Sequencial Candidato': 'candidate_sequence'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Setor econômico do doador': 'donor_economic_sector'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Setor econômico do doador originário': 'original_donor_economic_sector'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Sigla da UE': 'electoral_unit_abbreviation'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Sigla Partido': 'party_acronym'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Sigla UE doador': 'donor_electoral_unit_abbreviation'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Tipo doador originário': 'original_donor_type'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Tipo receita': 'revenue_type'}, inplace=True)
+donations_committees_concatenated.rename(columns={'UF': 'state'}, inplace=True)
+donations_committees_concatenated.rename(columns={'Valor receita': 'revenue_value'}, inplace=True)
```

Comment:
You could use a dictionary to handle multiple renames at once ; )

Reply:
jesus, that was a terrible mistake hahaha fixed!